packages/manager/modules/pci/src/projects/project/notebooks/dashboard/attach-data/attach-data.constants.js

```diff
@@ -0,0 +1,12 @@
+export const NOTEBOOK_ATTACH_DATA_INFO_LINK = {
+  DEFAULT: 'https://docs.ovh.com/fr/logs-data-platform/kubernetes-fluent-bit/',
+};
+
+export const NOTEBOOK_ATTACH_DATA_DATAGRID = {
+  MAX_ITEMS_PER_PAGE: 10,
```

Comment:
Don't know if it's really useful, this constant is used only in one place.
Would make more sens to have a common pagination for all datagrids (but it's another topic).

Reply:
Moved into controller.